### PR TITLE
[MBL-17567][Student][Teacher] Create edit todo tests

### PR DESCRIPTION
**Pull request by kristofnemere**
Test plan: I made some changes to overflow menu and single choice dialog, smoke test those please.

refs: MBL-17567
affects: Student, Teacher
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by kdeakinstructure — APPROVED** [review]

File: automation/espresso/src/main/kotlin/com/instructure/canvas/espresso/common/pages/compose/CalendarEventCreateEditPage.kt [hunk 1]
```diff
@@ -97,4 +97,15 @@ class CalendarEventCreateEditPage(private val composeTestRule: ComposeTestRule)
...
+
+    fun assertUnsavedChangesDialog() {
+        composeTestRule.onNodeWithText("Exit without saving?").assertIsDisplayed()
+        composeTestRule.onNodeWithText("Are you sure you would like to exit without saving?").assertIsDisplayed()
+        composeTestRule.onNodeWithText("Cancel").assertIsDisplayed()
+        composeTestRule.onNodeWithText("Exit").assertIsDisplayed()
+    }
+
+    fun clickClose() {
+        composeTestRule.onNodeWithTag("appBarNavigationIcon").performClick()
```
[the remark this answers is not in the document]
Comment (kristofnemere) [reply automation/espresso/src/main/kotlin/com/instructure/canvas/espresso/common/pages/compose/CalendarEventCreateEditPage.kt:109]: all test tags start with lowercase

File: libs/pandautils/src/main/java/com/instructure/pandautils/compose/composables/CanvasAppBar.kt [hunk 2]
```diff
@@ -51,7 +51,10 @@ fun CanvasAppBar(
         backgroundColor = colorResource(id = R.color.backgroundLightestElevated),
         contentColor = colorResource(id = R.color.textDarkest),
         navigationIcon = {
-            IconButton(onClick = navigationActionClick) {
+            IconButton(
+                onClick = navigationActionClick,
+                modifier = Modifier.testTag("appBarNavigationIcon")
```
Comment (tamaskozmer) [inline libs/pandautils/src/main/java/com/instructure/pandautils/compose/composables/CanvasAppBar.kt:56]: I think you can find the icon by content description, you don't need the tag here.

**Review by tamaskozmer — APPROVED** [review]

**Comment by inst-danger** [comment]
<p>Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Pull request merged**